ppml/base/Dockerfile

```diff
@@ -0,0 +1,125 @@
+ARG BIGDL_VERSION=2.2.0-SNAPSHOT
+ARG http_proxy
+ARG https_proxy
+ARG no_proxy
+
+FROM ubuntu:20.04
+
+ARG BIGDL_VERSION
+ENV BIGDL_VERSION                       ${BIGDL_VERSION}
+ENV LOCAL_IP                            127.0.0.1
+ENV SGX_MEM_SIZE                        32G
+ENV SGX_LOG_LEVEL                       error
+# TODO: decide whether we need this TINI_VERSION
+# ENV TINI_VERSION                        $TINI_VERSION
+ENV LC_ALL                              C.UTF-8
+ENV LANG                                C.UTF-8
+
+
+
+# TODO: decide whether we need these directories in gramine, or probably move to later images?
+# RUN mkdir -p /ppml/trusted-big-data-ml/work && \
+#     mkdir -p /ppml/trusted-big-data-ml/work/lib && \
+#     mkdir -p /ppml/trusted-big-data-ml/work/keys && \
+#     mkdir -p /ppml/trusted-big-data-ml/work/password && \
+#     mkdir -p /ppml/trusted-big-data-ml/work/data && \
+#     mkdir -p /ppml/trusted-big-data-ml/work/models && \
+#     mkdir -p /ppml/trusted-big-data-ml/work/apps && \
+#     mkdir -p /ppml/trusted-big-data-ml/work/examples/bigdl && \
+#     mkdir -p /root/.keras/datasets && \
+#     mkdir -p /root/.zinc && \
+#     mkdir -p /root/.m2 && \
+#     mkdir -p /root/.kube/
+#     mkdir -p /ppml/trusted-big-data-ml/fl
+
+RUN mkdir -p /ppml/trsuted-big-data-ml/
+
+ADD ./bash.manifest.template /ppml/trusted-big-data-ml/bash.manifest.template
+ADD ./Makefile /ppml/trusted-big-data-ml/Makefile
+ADD ./init.sh /ppml/trusted-big-data-ml/init.sh
+ADD ./clean.sh /ppml/trusted-big-data-ml/clean.sh
+
+# TODO: Three patches, do we really need this?
```

Comment:
I think we need these python patches
if not, the vfork will breaks python code on gramine

Reply:
Thanks, will add it to the Dockerfile